portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,124 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+
```

Comment:
Nit: I don't think you need these newlines. Html files generally include newlines to show separation between bigger components like between two divs.

Same comment for projects.html file